envpool/mujoco/dmc/mujoco_dmc_align_test.py

```diff
@@ -149,6 +153,12 @@ def test_pendulum(self) -> None:
       "pendulum", ["swingup"], DmcPendulumEnvSpec, DmcPendulumDMEnvPool
     )
 
+  def test_point_mass(self) -> None:
+    self.run_align_check_entry(
+      "point_mass", ["easy", "hard"], DmcPointMassEnvSpec,
+      DmcPointMassDMEnvPool
+    )
```

Comment:
For the reward test, it will fail. How to solve this fine accuracy problem between `1.64230036e-249` and `0`:
```cmd
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 1 / 1 (100%)
Max absolute difference: 1.64230036e-249
Max relative difference: inf
 x: array(1.6423e-249)
 y: array(0., dtype=float32)
```